.eslintrc

```diff
@@ -10,7 +10,8 @@
     "@typescript-eslint",
     "react",
     "react-hooks",
-    "jest"
+    "jest",
+    "unicorn"
```

Comment:
What is unicorn?

Reply:
Unicorn is the ESLint extension which allows us to do the expiring comments. Link here: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/expiring-todo-comments.md